dns/nextdns-community/src/etc/inc/plugins.inc.d/nextdns.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+
+
+function nextdns_enabled()
+{
+    $model = new \OPNsense\Nextdns\General();
+    return (string)$model->enabled == '1';
+}
+
+function nextdns_services()
+{
+    $services = array();
+
+    if (!nextdns_enabled()) {
+        return $services;
+    }
+
+    $services[] = array(
+        'description' => gettext('nextdns'),
+        'configd' => array(
+            'restart' => array('nextdns restart'),
+            'start' => array('nextdns start'),
+            'stop' => array('nextdns stop'),
+        ),
+        'name' => 'nextdns',
+        'pid' => '/var/run/nextdns.pid'
```

Comment:
Is there really a pidfile? If not, you'll not get  the icon.
Any chance to set one? Otherwise we could create a rc script and start via daemon

Reply:
From what I understand, .pid works as expected.
When nextdns is running:
![image](https://user-images.githubusercontent.com/31756078/118507596-3b151180-b6e3-11eb-8de9-695a331b2067.png)

When nextdns is stopped:
![image](https://user-images.githubusercontent.com/31756078/118507745-5ed85780-b6e3-11eb-82db-ee4876c74397.png)